util/ibex_config.py

```diff
@@ -152,28 +215,40 @@ def get_config_file_location():
 
 
 def main():
-    config_outputs = {'fusesoc_opts': _config_dict_to_fusesoc_opts}
+    outputters = [
+        FusesocOpts(),
+        SimOpts('vcs_opts', 'VCS compile',
+                lambda p, v: '-pvalue+' + p + '=' + v,
+                lambda d, v: '+define+' + d + '=' + v, '.'),
+        SimOpts('riviera_sim_opts', 'Riviera simulate',
+                lambda p, v: '-g/' + p + '=' + v, lambda d, v: None, '/'),
+        SimOpts('riviera_compile_opts', 'Riviera compile', lambda p, v: None,
+                lambda d, v: '+define+' + d + '=' + v, '/')
```

Comment:
This is quite hard to parse by eye because of the commas in the lambda syntax. Probably better with newlines between arguments.

Reply:
I blame the pylint auto fix